proto/system_config.proto

```diff
@@ -195,6 +195,9 @@ message SwitchSetup {
   // Interface name for external switch data plane
   string data_intf = 42;
 
+  // VXLAN configuration information
```

Comment:
Add this at the end, after the switch password config?

Reply:
done